### ISSUE TITLE
Minor issue with definition of ‘located in’

### ISSUE THREAD

**Issue by gregfowlerphd** [BFO-ontology/BFO-2020#103]
**Describe the bug**

The definition reads:

> b located in c =Def b is an independent continuant & c is an independent & neither is a spatial region & there is some time t such that the spatial region which b occupies at t is continuant part of the spatial region which c occupies at t

Notice that the word ‘continuant’ is missing after the second instance of ‘independent’.

**Expected behavior**

I believe 'continuant' should be added to the definition at the relevant place.

**Comment by alanruttenberg** [comment]
Agreed, but it seems to be a bug only in the out-of-sync BFO OWL. In the original source the definition is

> b located in c at t =Def. b and c are independent continuants and not spatial regions, and the spatial region which b occupies at t is a (proper or improper) continuant part of the spatial region which c occupies at t
